libs/react-helpers/src/lib/generic-data-provider.ts

```diff
@@ -248,6 +253,30 @@ function makeDataProviderInternal<
     callbacks.forEach((callback) => notify(callback, updateData));
   };
 
+  const call = (
```

Comment:
Please add unit tests to generic-data-provider.spec.ts with usage of errorPolicyGuard

Reply:
done